lib/api/apiUtils/object/versioning.js

```diff
@@ -142,44 +188,98 @@ export function versioningPreprocessing(bucketName, bucketMD, objectKey,
             if (nullVersionId === undefined) {
                 return callback(null, options);
             }
-            options.deleteNullVersionData = true;
             return _deleteNullVersionMD(bucketName, objectKey,
-                { versionId: nullVersionId }, log,
-                err => callback(err, options));
+                { versionId: nullVersionId }, log, (err, nullDataToDelete) => {
+                    if (err) {
+                        process.stdout.write('====================\n');
```

Comment:
I added these in a dropme commit just to print more information in an end-to-end. A previous end-to-end failed with InternalError and I couldn't find the relevant logs (not sure what log level the end to ends run with). Unfortunately, I couldn't duplicate the error so I dropped the commit.